pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/StartServiceManagerCommand.java

```diff
@@ -205,9 +206,9 @@ public boolean execute()
     }
   }
 
-  private String startServiceManager() {
+  private String startServiceManager(boolean bootstrapMode) {
```

Comment:
Do we need this extra flag? Seems the `bootstrapMode` can be derived from the `_port`?